content/recipes/sql-sequelize.md

```diff
@@ -1,6 +1,6 @@
 ### SQL (Sequelize)
 
-##### This chapter applies only to TypeScript
+#### This chapter applies only to TypeScript
```

Comment:
This is intentional (styling + margins) :)

Reply:
undo ;)